src/transform/EditTransform.js

```diff
@@ -63,12 +72,86 @@ const newEditSectionHeader = (document, index, level, titleHTML, showEditPencil
     const button = newEditSectionButton(document, index)
     element.appendChild(button)
   }
-
   return element
 }
 
+/**
+ * Elements needed to show or add page title description.
+ * @param {!Document} document
+ * @param {?string} titleDescription Page title description.
+ * @param {?string} addTitleDescriptionString Localized string e.g. 'Add title description'.
+ * @param {?boolean} isTitleDescriptionEditable Whether title description is editable.
+ * @return {?HTMLElement}
+ */
+const titleDescriptionElements = (document, titleDescription, addTitleDescriptionString,
+  isTitleDescriptionEditable) => {
+  const descriptionExists = titleDescription !== undefined && titleDescription.length > 0
+  if (descriptionExists) {
+    const p = document.createElement('p')
+    p.id = IDS.TITLE_DESCRIPTION
+    p.innerHTML = titleDescription
+    return p
+  }
+  if (isTitleDescriptionEditable) {
+    const a = document.createElement('a')
+    a.href = '#'
+    a.setAttribute(DATA_ATTRIBUTE.ACTION, ACTION_ADD_TITLE_DESCRIPTION)
+    const p = document.createElement('p')
+    p.id = IDS.ADD_TITLE_DESCRIPTION
+    p.innerHTML = addTitleDescriptionString
+    a.appendChild(p)
+    return a
```

Comment:
Is it possible to let the `p` to be the parent and make `p.appendChild(a)` and then return `p`? If so, we can set the `dir` attritube by selecting the `p` in any condition.

Also, if changes the logic above, you might need to remove the `display: inline` from `#pagelib_edit_section_add_title_description` to make the `dir` works.

Reply:
Oh can you double-check that you still need to set `dir` - if I recall correctly Android sets `dir` on the `html` tag so you probably don't need to set it on nested elements such as this one...

Reply:
(I think we only set `dir` on the `html` tag on iOS these days...)

Reply:
Thanks @montehurd !
We have been set the `dir` tag to the `div` tag that has the class name `pagelib_edit_section_header`. 

But looks like set `dir` in the parent `div` (the `id='content'`) is a good idea and much simpler!